zegami_sdk/annotation.py

```diff
@@ -30,8 +30,10 @@ def __init__(self, collection, annotation_data, source=None):
         """
 
         self._collection = collection  # Collection instance
-        self._source = source  # Source instance
-        self._data = annotation_data  # { imageset_id, image_index, type, annotation }
+        self._imageset_id = collection._get_imageset_id(source=source)
+        self._row_index = row_index
+
+        self._data = collection.fetch_annotation_data(self._row_idx)  # { imageset_id, image_index, type, annotation }
```

Comment:
As things stand, annotations apply to the specific image (i.e. source) So the same row could have an annotation for the image of one source, but not for the other. The same annotation could even be associated to multiple rows if multiple rows refer to the same image.
There are clearly scenarios where it'd make more sense for the annotation to transcend the sources, but also cases where it doesn't.

Not sure how best to deal with that, perhaps include source in `fetch_annotation_data`?

Reply:
In fact, I don't see `fetch_annotation_data` in collection.py. Is there a change missing from this PR?

Reply:
Not sure how this creeped in, my version doesn't even use this any more! New commit fixes